components/layout/navbar/search.tsx

```
@@ -1,21 +1,25 @@
-'use client';
+"use client";
 
-import { MagnifyingGlassIcon } from '@heroicons/react/24/outline';
-import Form from 'next/form';
-import { useSearchParams } from 'next/navigation';
+import { MagnifyingGlassIcon } from "@heroicons/react/24/outline";
+import Form from "next/form";
+import { useSearchParams } from "next/navigation";
 
 export default function Search() {
   const searchParams = useSearchParams();
 
   return (
-    <Form action="/search" className="w-max-[550px] relative w-full lg:w-80 xl:w-full">
+    <Form
+      action="/search"
+      prefetch={false}
```

Was prefetch breaking search?

---

Yea I was going to bring this one up with you - bit of a head-scratcher. Search works fine locally, but when deployed, without setting `prefetch={false}` it just doesn't want to move to the /search route. After failing silently with no errors in client logs or deployment logs - you'll also find that no links work at all. Refreshing the page fixes the links, but once you try search, the current page just stops responding.